### PR TITLE
child dashboard component

### PR DESCRIPTION
**Pull request by Schrese**
It can be linked to from the home page for now

# Description
This is the child dashboard feature. The purpose of this is to set up the dashboard for when we are ready to begin passing state around to the various components. 
Fixes #

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [x] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by Nick-Ohman — DISMISSED** [review]
It all looks good, i like the alt tags.

**Review by SandraCoburn — DISMISSED** [review]
Nice! Thanks for the adding the header-text class.

**Review by Schrese — COMMENTED** [review]
The index shouldn't have changed, but if it did I can fix it

File: src/styles/ChildDashboard.less [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.dash-container {
+  height: 73vh;
+}
+
+.ant-row {
+  height: 50%;
+}
+
+.accept-mission {
+  background: url('../assets/images/child_dashboard_images/accept_the_mission.svg')
```
Comment (bramirez96) [inline src/styles/ChildDashboard.less:13]: :')

**Review by JLC6290 — APPROVED** [review]

**Review by bramirez96 — APPROVED** [review]
Looks great!

**Pull request merged**